src/Draft.js

```diff
@@ -43,7 +42,6 @@ const DraftPublic = {
   EditorState,
 
   CompositeDecorator: CompositeDraftDecorator,
-  Entity: DraftEntity,
   EntityInstance: DraftEntityInstance,
```

Comment:
I actually don't think so. I also noticed that I have a method `contentState.addEntity`, which takes a `DraftEntityInstance` but is actually not used... I'm not quite sure why I still have it but it might be a relict from porting over the methods of `DraftEntity` 1:1. Do you think we still need it for something?